File: nekkochat.client/src/Schemas/RegisterSchemas.ts
export default class LoginSchemas {
    public email: string;
    public fname: string;
    public lname: string;
    public password: string;
    public confirmpassword: string;
    public phoneNumber: string;
    public remember: boolean;
    public profilePhotoUrl: string;
    public about: string;
    constructor(
        email: string,
        fname: string,
        lname: string,
        password: string,
        confirmpassword: string,
        phoneNumber: string,
        profilePhotoUrl: string,
        about: string,
        remember: boolean,
    ) {
        this.email = email;
        this.fname = fname;
        this.lname = lname;
        this.password = password;
        this.confirmpassword = confirmpassword;
        this.phoneNumber = phoneNumber;
        this.remember = remember;
        this.profilePhotoUrl = profilePhotoUrl;
        this.about = about;
    }

}

File: NekkoChat.Server/Schemas/RegisterSchemas.cs
namespace NekkoChat.Server.Schemas
{
    public class RegisterSchemas
    {
        public string username { get; set; } = string.Empty;
        public string email { get; set; } = string.Empty;
        public string fname { get; set; } = string.Empty;
        public string lname { get; set; } = string.Empty;
        public string password { get; set; } = string.Empty;
        public string confirmPassword { get; set; } = string.Empty;
        public string phoneNumber { get; set; } = string.Empty;
        public string about { get; set; } = "Available";
        public string profilePhotoUrl { get; set; } = "/src/assets/avatar.png";
    }
}
